x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/expression/function/aggregate/Rate.java

```diff
@@ -168,16 +168,13 @@ long unitInMillis() {
     }
 
     @Override
-    public AggregatorFunctionSupplier supplier(List<Integer> inputChannels) {
-        if (inputChannels.size() != 2 && inputChannels.size() != 3) {
```

Comment:
This assertion can't be done now, unless we manually change the autogenerated function suppliers.
I don't think it's "required" though